Jenkinsfile

```diff
@@ -47,10 +47,12 @@ podTemplate(
                 sh("pip3 install poetry")
             }
             stage('Install all dependencies') {
+                sh("pyenv install 3.6.6 && pyenv install 3.7.2")
```

Comment:
I thought the we are using the multi-python docker image for this that comes with the right python versions already installed. If you already checked that, ignore this comment :)

Reply:
https://github.com/cognitedata/multi-python/blob/master/Dockerfile
    pyenv install 3.5.0 && \
    pyenv install 3.6.6 && \
    pyenv install 3.7.2

Reply:
Nice find, I did not notice that.